dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -20,17 +11,14 @@ def new
   # POST /plc/user_course_enrollments.json
   def create
     if @user_course_enrollment.save
-      redirect_to @user_course_enrollment, notice: 'User course enrollment was successfully created.'
```

Comment:
Eh - I didn't think so. The next iteration of the course view dashboard may include some one-time text that introduces you to the contents of the view, and that'll accomplish the same thing